.changeset/miip-color-sample-docs.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nl-design-system-candidate/color-sample-docs': minor
```

Comment:
I don't think this is a  minor change, but a patch. There is no new functionality, file or documentation and going back after this version wouldn't impact you very much.

```suggestion
'@nl-design-system-candidate/color-sample-docs': patch
```